Python/ceval.c

```diff
@@ -1644,15 +1652,21 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
     if (PyDTrace_FUNCTION_ENTRY_ENABLED())
         dtrace_function_entry(f);
 
+    if (!PyCodeObject_IsWarmedUp(co)) {
+        PyCodeObject_IncrementWarmup(co);
```

Comment:
I was worried about wraparound here, but it seems that's handled in _Py_Quicken(). Maybe that's worth a mention here?